contracts/vesting-lp/src/lib.rs

```diff
@@ -0,0 +1,5 @@
+pub mod contract;
+mod msg;
```

Comment:
```suggestion
pub mod msg;
```
This will allow me to import your `QueryMsg` from neutron-dao repo.